### ISSUE TITLE
Comment navigation arrows are broken

### ISSUE THREAD

**Issue by joytrekker** [ccrama/Slide#1148]
- [x] When using the comment navigation arrows the up arrow does not work if you press it after pressing the down arrow. If you scroll the comments a few pixels then the arrow works as expected. 
- [x] The arrows also jump to the wrong position if the top bar is auto hiding. If the top bar is auto hiding then the the nav arrows jump to the screen position as if the bar was visible. 
- [x] If you press the up arrow when viewing a top level comment then the screen jumps back to the top of the same comment instead of jumping to the previous comment. 

Slide version: v4.6.2
Android version: Marshmallow

**Comment by ccrama** [comment]
Cannot reproduce, I totally redid the code for this with the latest alpha. Can you post your version code here (just tap on the version number in Settings > About to copy it)? Thanks!

**Comment by joytrekker** [comment]
I compiled it yesterday. I'll try it again today.
Slide v4.6.2-94-7d2688a

**Comment by joytrekker** [comment]
I just recompiled and issue #1 seems to have gone away. However issues 2 & 3 are still there. I cleared the cache and installed the Play Store version as well but got the same results.

**Comment by ccrama** [comment]
2 and 3 are on purpose (it adds the height of the toolbar on purpose), and it jumps to the first instance of the parent comment above it that is top level. For 2, I just made it always show the toolbar if it's hidden

**Comment by joytrekker** [comment]
Issue #1 is back. Actually I don't think it ever went away. The padding fix only works for the down arrow.

Another issue
I noticed that if you keep pressing the down arrow sometimes it will work as expected for a few top level comments say 3-5 then for some reason it cycles back to the first top level comment. I think the arrow navigation need some review it has been all messed up since the animation was removed.
